web/package.json

```diff
@@ -86,12 +89,12 @@
     "react-icons": "^4.7.0",
     "react-map-gl": "^7.0.19",
     "react-router-dom": "6.14.2",
-    "react-spinners": "^0.13.6",
-    "react-spring-bottom-sheet": "3.5.0-alpha.0",
-    "recharts": "^2.1.16",
+    "react-spinners": "^0.13.8",
+    "react-spring-bottom-sheet": "3.4.1",
```

Comment:
That added back a lot of the duplicate packages I was hoping to get rid off 😐.

Hopefully it will resolve it self when we get all the related packages updated.

Reply:
I just discovered https://panejs.com/ yesterday which we might be able to use instead! :)

Reply:
Let's merge this and then look into that! Sadly it seems like the one we use now is no longer maintained.